middleware/middleware-1.3.14.md

```diff
@@ -0,0 +1,80 @@
+
+## Bug fix: It was technically possible to operate a stopped Queue
+Due to an issue in the Middleware, it was technically still possible to send receipts to a stopped Queue. However, since the TSE is usually deactivated in this case (if the respective flag is not used), this could lead to exceptions.
+
+With this fix, the Queue now correctly responds with the _ftState_ `0x4445000000000001` after it was deactivated, and doesn't contact the SCU and therefore the TSE anymore.
+
+## Bug fix: Launcher sometimes fails to start when operations take too long
+Due to a default limitation in Windows services, services must not take more than 30 seconds to start, otherwise they'll be flagged as "not responding" by Windows. This issue infrequently happens with components that need to download 3rd party components during the first initialization (e.g. Swissbit Cloud/Deutsche Fiskal), which may take more than 30 seconds with slow internet connections.
+
+This issue is now resolved, as the Launcher automatically requests more startup time starting from this version.
+
+## Bug fix: Diebold Nixdorf SCU throws exception during self test
+Due to unexpected behavior of Diebold Nixdorf TSEs with older firmware versions, the Middleware could run into issues when querying the TSE's current state details. As this operation is executed frequently during the self test of the TSE, this could lead to recurrent issues.
+
+The Middleware was updated to be compatible with the respective firmware versions.
+
+## Dependency update: Fiskal Cloud Connector v3 (Swissbit Cloud & Deutsche Fiskal)
+On February 15, version 3.0 of the _Fiskal Cloud Connector_ was released. This 3rd party background service is required both by the Swissbit Cloud and the Deutsche Fiskal Cloud TSEs, as it contains their local SMAERS application (and is therefore part of the TSE). When using the respective SCUs with versions >= 1.3.14, version 3 is automatically installed and initialized.
+
+Due to this change, Middleware instances using Swissbit Cloud SCUs now don't require Admin privileges anymore (unless e.g. an Admin port range is used, etc.). However, to automatically add the required Firewall exceptions, Admin privileges are still required. These Firewall exceptions can also be set manually, or - in user mode - via the respective popup. Please refer to our [documentation](TODOOOOOO) for more details.
+
+<div class="alert alert--warning" role="alert">Due to regulatory reasons caused by the updated TR-03153 certifications of Swissbit Cloud and Deutsche Fiskal TSEs, it's <b>not possible to update existing installations that are currently using v2 FCCs</b>. This only affects a very small amount of our partners, whom we will directly contact in the next days. SCUs that were not rolled our yet are not affected by this problem and have been automatically updated to use the latest FCC version.</div>
```

Comment:
Do we have a link that we can add to redirect them to the vendors?

Reply:
Unfortunately not, all of this is written to their internal docs only..